eureka2-read-server/src/main/java/com/netflix/eureka2/server/FullFetchInterestClientProvider.java

```diff
@@ -22,12 +22,15 @@
 import com.netflix.eureka2.server.config.EurekaCommonConfig;
 import com.netflix.eureka2.server.interest.FullFetchBatchingRegistry;
 import com.netflix.eureka2.server.interest.FullFetchInterestClient;
+import com.netflix.eureka2.transport.TransportClient;
 
 /**
  * @author Tomasz Bak
  */
 public class FullFetchInterestClientProvider implements Provider<FullFetchInterestClient> {
 
+    private static final String EUREKA_READ_CLIENT_ID = "eurekaReadClient";
```

Comment:
We might use the same class in different contexts. In this particular case it would work, but might not work in the other. I propose it keep it as an arbitrary name. In Eureka client it is even possible to set it explicitly via transport config.